### PR TITLE
Create operator CSV file

### PR DESCRIPTION
**Pull request by rthallisey**
- Start using images from https://quay.io/organization/cincinnati
- First pass at a CSV file

### CODE REVIEW

File: deploy/olm-catalog/cincinnati-operator/0.0.1/cincinnati-operator.v0.0.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        serviceAccountName: cincinnati-operator
+    strategy: deployment
+  installModes:
```
Comment (rthallisey) [inline deploy/olm-catalog/cincinnati-operator/0.0.1/cincinnati-operator.v0.0.1.clusterserviceversion.yaml:200]: Not sure what combination we want to go with here.  The OperatorGroup stuff always confuses me.

File: deploy/olm-catalog/cincinnati-operator/0.0.1/cincinnati-operator.v0.0.1.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+            "gitHubRepo": "cincinnati-graph-data",
+            "registry": "quay.io",
+            "replicas": 1,
+            "repository": "openshift-release-dev/ocp-release"
+          }
+        }
+      ]
+    capabilities: Basic Install
+    description: Creates and maintains a Cincinnati instance
+    operatorframework.io/suggested-namespace: cincinnati
```
Comment (djzager) [inline deploy/olm-catalog/cincinnati-operator/0.0.1/cincinnati-operator.v0.0.1.clusterserviceversion.yaml:25]: does this deserve an `openshift-cincinnati` namespace?

File: deploy/olm-catalog/cincinnati-operator/0.0.1/cincinnati-operator.v0.0.1.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+                name: cincinnati-operator
+            spec:
+              containers:
+              - command:
+                - cincinnati-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
```
Comment (djzager) [inline deploy/olm-catalog/cincinnati-operator/0.0.1/cincinnati-operator.v0.0.1.clusterserviceversion.yaml:96]: I have no legit reason for using `targetNamespaces` to make me nervous. It just does. Any specific reason not to use `metadata.name`?

File: deploy/olm-catalog/cincinnati-operator/0.0.1/cincinnati-operator.v0.0.1.clusterserviceversion.yaml (hunk 1 above)
Comment (djzager) [reply deploy/olm-catalog/cincinnati-operator/0.0.1/cincinnati-operator.v0.0.1.clusterserviceversion.yaml:200]: I think the primary advantage of setting `AllNamespaces: true` is that it prevents the operator from being installed more than 1x (assuming it was first subscribed to watching all namespaces). Don't see a specific reason to mark it false. Everything else about these installModes looks legit.

File: deploy/operator.yaml [hunk 4]
```diff
@@ -15,8 +15,7 @@ spec:
       serviceAccountName: cincinnati-operator
       containers:
         - name: cincinnati-operator
-          # TODO Replace this with the built image name
-          image: docker.io/mhrivnak/cincinnati-operator:latest
+          image: quay.io/cincinnati/cincinnati-operator:latest
```
Comment (mhrivnak) [inline deploy/operator.yaml:18]: \o/

File: deploy/olm-catalog/cincinnati-operator/0.0.1/cincinnati-operator.v0.0.1.clusterserviceversion.yaml (hunk 2 above)
Comment (rthallisey) [reply deploy/olm-catalog/cincinnati-operator/0.0.1/cincinnati-operator.v0.0.1.clusterserviceversion.yaml:25]: I has that originally, but my community instincts made me change it.  OpenShift seems to be the only consumers of Cincinnati, so it probably make sense to have the openshift prefix.

File: deploy/olm-catalog/cincinnati-operator/0.0.1/cincinnati-operator.v0.0.1.clusterserviceversion.yaml (hunk 3 above)
Comment (rthallisey) [reply deploy/olm-catalog/cincinnati-operator/0.0.1/cincinnati-operator.v0.0.1.clusterserviceversion.yaml:96]: No reason; a bunch of this is boiler plate.

**Comment by rthallisey** [comment]
Any more comments?  Would like to merge this in.

**Pull request merged**